addons/stock_landed_costs_mrp/views/stock_landed_cost_views.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo><data>
+    <record id='view_stock_landed_cost_mrp_form' model='ir.ui.view'>
+        <field name="name">stock.landed.cost.mrp.form</field>
+        <field name="model">stock.landed.cost</field>
+        <field name="inherit_id" ref="stock_landed_costs.view_stock_landed_cost_form"/>
+        <field name="arch" type="xml">
+            <field name="target_model" position="attributes">
+                <attribute name="invisible">0</attribute>
+                <attribute name="groups">mrp.group_mrp_user</attribute>
```

Comment:
groups won't work if tomorrow we add repair_landed_costs

Reply:
Well, I don't see any easy solution for now :/